src/metabase/models/dashboard_card_series.clj

```diff
@@ -12,3 +13,21 @@
 
 (doto :model/DashboardCardSeries
   (derive :metabase/model))
+
+;; Serialization
+
+(defmethod serdes/generate-path "DashboardCardSeries" [_ _] nil)
```

Comment:
Just had a thought that it would be really nice to have a way to enforce the fact that `serdes/generate-path` should return `nil` if the entity is nested and has no entity_id.

At the very least, we might want to explain this somewhere like here, or in the `serdes/generate-path` docstring:
https://github.com/metabase/metabase/blob/7491359edcdcb8d6f824c08a61ad9ea87542e6f1/src/metabase/models/serialization.clj#L195-L199